config/prometheus/monitor.yaml

```diff
@@ -17,8 +17,8 @@ spec:
       tlsConfig:
         insecureSkipVerify: true
     - path: /metrics
-      port: lvms-metrics
-      scheme: http
+      port: https
```

Comment:
Is the servicemonitor via tls supported normally or is there any config change we need to do?